snuba/web/rpc/v1/resolvers/R_eap_spans/common/common.py

```diff
@@ -108,13 +111,24 @@ def use_eap_items_table(request_meta: RequestMeta) -> bool:
     if settings.USE_EAP_ITEMS_TABLE:
         return True
 
+    use_eap_items_orgs = state.get_config("use_eap_items_orgs")
+    if use_eap_items_orgs:
+        use_eap_items_orgs = map(int, use_eap_items_orgs.strip("[]").split(","))
+
     use_eap_items_table_start_timestamp_seconds = state.get_int_config(
         "use_eap_items_table_start_timestamp_seconds"
     )
+
     if (
         state.get_config("use_eap_items_table", False)
         and use_eap_items_table_start_timestamp_seconds is not None
     ):
+        if (
+            use_eap_items_orgs
+            and request_meta.organization_id not in use_eap_items_orgs
+        ):
+            return False
+
```

Comment:
this logic is a bit weird. You'd have to eventually put in every org id to turn it on for everyone. I think instead it should be:

1. `use_eap_items_table_all` - turned on for everyone
2. `turned_on_for_org`-  `request_meta.organization_id in use_eap_items_orgs`
3. `within_items_timerange` - aka ` request_meta.start_timestamp.seconds  >= use_eap_items_table_start_timestamp_seconds` 

`return within_items_timerange and (use_eap_items_table_all or turned_on_for_org)`